alf/algorithms/td_loss.py

```diff
@@ -31,9 +31,20 @@ class TDLoss(nn.Module):
     def __init__(self,
                  gamma: Union[float, List[float]] = 0.99,
                  td_error_loss_fn: Callable = element_wise_squared_loss,
+                 clip: float = 0.,
                  td_lambda: float = 0.95,
                  normalize_target: bool = False,
                  debug_summaries: bool = False,
+                 lb_target_q: float = 0.,
+                 default_return: float = -1000.,
+                 improve_w_goal_return: bool = False,
+                 improve_w_nstep_bootstrap: bool = False,
+                 improve_w_nstep_only: bool = False,
+                 lower_bound_constraint: float = 0.,
+                 lb_loss_scale: bool = False,
+                 reward_multiplier: float = 1.,
+                 positive_reward: bool = True,
+                 use_retrace: bool = False,
```

Comment:
To me, it's not clear which of these arguments are really essential to `TDLoss` in a general sense. Some of them seem only applicable to specific scenarios.